spark/src/test/resources/tpcds-micro-benchmarks/char_type.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
```

Comment:
how this test related to rpad? 🤔

Reply:
They are related as their schema types are CHAR()